cylc/flow/task_remote_cmd.py

```diff
@@ -51,6 +52,8 @@ def remote_init(uuid_str, rund, indirect_comm=None):
     os.makedirs(rund, exist_ok=True)
     oldcwd = os.getcwd()
     os.chdir(rund)
+    # Extract job.sh from library, for use in job scripts.
+    get_pkg_resources(SuiteSrvFilesManager.DIR_BASE_SRV, ['etc/job.sh'])
```

Comment:
I was wondering why the return value of `get_...` was not being used, then realized it is actually writing the resource to a path. Would it be helpful to call the method `extra_pkg_resources`? (simply suggested the first word in the method docstrings that I just read)

Reply:
Good point, I'll do that.